app/models/dashboard.rb

```diff
@@ -3,7 +3,25 @@
 # Summary of recent activity across the whole app
 class Dashboard
   def recent_uploads_by_provider
-    Upload.recent.where('created_at > ?', 30.days.ago).group_by(&:organization)
+    recent_uploads_by_provider = Upload.recent.where('created_at > ?', 30.days.ago).group_by(&:organization)
+    # Find any providers that didn't have uploads in the last 30 days
+    recent_inactive_orgs = Organization.providers - recent_uploads_by_provider.keys
+
+    # Add inactive providers into the returned hash
+    # Populate their last upload with upload outside the 30 day window, if one exists
+    # Otherise, if no uploads ever, set value to nil
+    # Also set a recently_inactive flag to true for use in the display
+    recent_inactive_orgs.each do |org|
+      org.recently_inactive = true
+      recent_uploads_by_provider.merge!({ org => inactive_org_upload_value(org) })
+    end
+
+    recent_uploads_by_provider
```

Comment:
Maybe instead of combining these into a single hash, return them as part of an array or hash to be handled separately as needed by the view. Untested, but something like:
```suggestion
    uploads_by_inactive_provider = recent_inactive_orgs.to_h { |org| [org, inactive_org_upload_value(org)] }
    { active_orgs: recent_uploads_by_provider, inactive_orgs: uploads_by_inactive_provider  }
```